plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchColumnHandle.java

```diff
@@ -58,6 +66,13 @@ public boolean isSupportsPredicates()
         return supportsPredicates;
     }
 
+    private static final List<Type> NUMERIC_TYPES = Arrays.asList(REAL, DOUBLE, TINYINT, SMALLINT, INTEGER, BIGINT);
+
+    public boolean isNumericType()
```

Comment:
This method doesn't really belong here. Move it to the MetricAggregation class, which is the only one that cares about it for now.